algorithms/createStr.js

```diff
@@ -0,0 +1,12 @@
+
+const createStr =(word,num, i=0)=>{
+  let emptyStr = " ";
+  if(i==num){
+    return emptyStr; 
+  } else{
+   console.log(word);
+    i=i+1;
+    createStr(word, num, i);
+  }
+}
+createStr('hello',3);
```

Comment:
result: `hellohellohello`

If I call `createStr('s2', 3);` I should get `s2s2s2`